beacon-chain/rpc/service_test.go

```diff
@@ -34,3 +34,36 @@ func TestBadEndpoint(t *testing.T) {
 	rpcService.Stop()
 	testutil.AssertLogsContain(t, hook, "Stopping service")
 }
+
+func TestInsecureEndpoint(t *testing.T) {
+	hook := logTest.NewGlobal()
+	rpcService := NewRPCService(context.Background(), &Config{Port: "9999"})
+
+	rpcService.Start()
+
+	testutil.AssertLogsContain(t, hook, "Starting service")
+	testutil.AssertLogsContain(t, hook, fmt.Sprintf("RPC server listening on port :%s", rpcService.port))
+	testutil.AssertLogsContain(t, hook, "You are using an insecure gRPC connection")
+
+	rpcService.Stop()
+	testutil.AssertLogsContain(t, hook, "Stopping service")
+}
+
+func TestRPCMethods(t *testing.T) {
+	s := &Service{}
+	if _, err := s.FetchShuffledValidatorIndices(context.Background(), nil); err == nil {
+		t.Error("Wanted error: unimplemented, received nil")
```

Comment:
This is OK, but a better pattern is to check that the error returned was the error you expect. 

This assertion is that any error returns.

Reply:
This will be changed very soon once we implement the logic of the beacon RPC server